Security/day03/Medium/Challenge-1/docker-compose.yml

```diff
@@ -0,0 +1,4 @@
+version: '3'

```

Comment:
useless file ?

Reply:
delete all challenges and just give a readme to give the the url of pathwar